src/bridge.ts

```diff
@@ -1212,6 +1212,19 @@ export class Bridge {
         return true;
     }
 
+    /**
+     * Close the appservice HTTP listener, and clear all timeouts.
+     * @returns Resolves when the appservice HTTP listener has stopped
+     */
+    public async close() {
+        if (this.intentLastAccessedTimeout) {
+            clearTimeout(this.intentLastAccessedTimeout);
+        }
+        if (this.appservice) {
+            await this.appservice.close();
+        }
+    }
```

Comment:
I think we should reset these or have a boolean flag that prevents the Bridge from being reused.
```suggestion
        if (this.intentLastAccessedTimeout) {
            clearTimeout(this.intentLastAccessedTimeout);
            this.intentLastAccessedTimeout = null;
        }
        if (this.appservice) {
            await this.appservice.close();
            this.appservice = null;
        }
    }
```

Reply:
Yeah, works for me.